filters/all.js

```diff
@@ -340,12 +340,7 @@ function fixType([name, javaName, property]) {
       // Picking a type for the user may be difficult - especially since the first avro union value must be the default value which is normally of type null.
       typeName = 'Object';
     } else {
-      // check to see if it's a ref to another schema.
-      typeName = property.ext('x-parser-schema-id');
-
-      if (!typeName) {
-        throw new Error(`Can't determine the type of property ${  name}`);
-      }
+      throw new Error(`Can't determine the type of property ${  name  }`);
```

Comment:
We could assume it's an object and the class name derived from the model class mapping is correct with
`typeName = javaName;` but it may not always be true. The model class mapping takes the x-schema-parser-id into account when determining the class name so `typeName = javaName` here should be the same idea. However, we'd only do this if we're okay with guessing the type.
If we do guess, this error here would never be thrown because (I think) there is always an x-parser-schema-id present.